docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md

```diff
@@ -202,3 +202,76 @@ Then `./gradlew setup` must be executed.
 !!! warning
     It is mandatory to execute this step before installing the module. 
 
+
+## EAN 128
+
+### Overview
+
+It is a module for the transmission of information between the agents of the supply chain under the specifications of the Code 128 barcode. This, provide generic Utils for EAN128 Definition and parsing
+
+Is a Java-based module for managing EAN-128 barcodes in an Etendo ERP context. Below is an overview of what each file appears to do:
+
+### SelectorFieldPropertyDataSource.java
+
+**Purpose**: Implements a data source for a custom field selector.
+**Main Operations**:Verifies whether a user has the access right to fetch data from the datasource.
+Filters out Blob and OneToMany properties when getting entity properties.
+
+### EAN128Utils.java
+
+**Purpose**: Utility class for handling EAN-128 data parsing and manipulation.
+
+**Main Operations**:Defines standard date formats and JSON keys.
+Parses incoming EAN128 Data JSON and extracts relevant information such as lot, serial number, expiration dates, etc.
+If Inventory Line quantity was parsed, it modifies the relevant BaseOBObject's quantity.
+Provides methods to edit OpenBravo base objects (BaseOBObject) based on the parsed EAN-128 data.
+
+### EANType.java
+
+**Purpose**: Web Service to manage EAN-128 Types.
+
+**Main Operations**:Handles HTTP GET requests and retrieves EAN-128 type information.
+The information fetched depends on the warehouse specified in the request parameters.
+If a warehouse-specific EAN-128 type isn't available, it attempts to fetch the default one.
+Converts EAN-128 type to a JSON object and returns it as a response.
+
+**Key Libraries used**:
+- Etendo DAL for database interaction.
+- Apache Commons Lang for String manipulations.
+- JSON and Jettison for JSON parsing.
+
+### SMFEANComponentProvider.java
+
+**Purpose**: It defines the global resources that are necessary for the operation of the components it provides.
+
+## Mobile Scan
+
+### Overview
```

Comment:
All modules have their Technical Aspects description and in this case only Overview and description of methods or classes are added.